errors/errors.go

```diff
@@ -20,6 +20,12 @@ type TraceableError struct {
 	What  string
 }
 
+// ValidationError is a type of error
```

Comment:
Please add a comment explaining why we have a ValidationError type, and when it should be used.

Reply:
done